### PR TITLE
Fix/update links to dbt learn

### PR DESCRIPTION
**Pull request by coapacetic**
## What are you changing in this pull request and why?
The training and web team recently launched the new LMS platform over at learn.getdbt.com. This is a PR to start the process of updating all of the courses.getdbt.com/ links to learn.getdbt.com.

## Checklist
- [X] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [X] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [x] Double check that all modified links actually work -- battle test it
- [x] Get full approval from the web team to hit send on this 🚀

### CODE REVIEW

**Comment by mirnawong1** [comment]
thanks @coapacetic  this looks great and the links word for me in the [staging link](https://docs-getdbt-nlw1uruao-dbt-labs.vercel.app/guides/core-to-cloud-3?step=9) 

❓ i think it might also be beneficial to add a redirect here: https://github.com/dbt-labs/docs.getdbt.com/blob/current/website/vercel.json as well. _however_, i believe we have reached our limit on adding redirects. i don't want to block this pr but wanted to flag this in case we can also raise that in a related ticket since you're working with the web team on this. cc @runleonarun @matthewshaver @nghi-ly 

also separately, i believe the SL tableau integration is ga now so [this course page](https://learn.getdbt.com/courses/tableau-querying-the-semantic-layer) might need the 'beta' removed.
Screenshot 2024-05-23 at 11 15 41

**Comment by coapacetic** [comment]
Thanks @mirnawong1 -- appreciate the super quick review! 🙌 

re: Tableau BETA course -- great catch, I'll loop in @MichelleBaird-DBT to tackle that one.

re: redirects, I'll yield to the docs / web team on this one with the deeper understanding of how we use redirects.

related: I'll make sure we sync with Joey on migration tactics here so we merge this at the right time.

🚀

**Comment by mirnawong1** [comment]
you got it! will revert back!

**Comment by mirnawong1** [comment]
@coapacetic i'll ask the web team and can you let us now when you'd like this merged? thank you!

**Comment by coapacetic** [comment]
@mirnawong1 sounds great on the redirects -- I'll swing back here on when we actually hit the merge button.  Also tagged you in a slack thread for viz

**Comment by joeygaultdbt** [comment]
@coapacetic My recommendation here is to manually change the links within the content instead of using the redirect file while we figure out how to clean that one up and make it more efficient. This work will likely take a few weeks and I don't want to block your LMS work.

**Review by mirnawong1 — APPROVED** [review]
🚢 hey @coapacetic , approving this pr for you and the team to merge when you and @joeygaultdbt chat about the timeline

File: website/docs/guides/adapter-creation.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ The more you can answer Yes to the below questions, the easier your adapter deve
 
 ### Training
 
-- the developer (and any product managers) ideally will have substantial experience as an end-user of dbt. If not, it is highly advised that you at least take the [dbt Fundamentals](https://courses.getdbt.com/courses/fundamentals) and [Advanced Materializations](https://courses.getdbt.com/courses/advanced-materializations) course.
+- the developer (and any product managers) ideally will have substantial experience as an end-user of dbt. If not, it is highly advised that you at least take the [dbt Fundamentals](https://learn.getdbt.com/courses/dbt-fundamentals) and [Advanced Materializations](https://learn.getdbt.com/courses/advanced-materializations) course.
```
Comment (matthewshaver) [inline website/docs/guides/adapter-creation.md]:
```suggestion
- The developer (and any product managers) ideally will have substantial experience as an end-user of dbt. If not, it is highly advised that you at least take the [dbt Fundamentals](https://learn.getdbt.com/courses/dbt-fundamentals) and [Advanced Materializations](https://learn.getdbt.com/courses/advanced-materializations) course.
```

File: website/docs/guides/redshift-qs.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ In this quickstart guide, you'll learn how to use dbt Cloud with Redshift. It wi
 - Schedule a job to run
 
 :::tips Videos for you
-You can check out [dbt Fundamentals](https://courses.getdbt.com/courses/fundamentals) for free if you're interested in course learning with videos.
+You can check out [dbt Fundamentals](https://learn.getdbt.com/courses/dbt-fundamentals) for free if you're interested in course learning with videos.
```
Comment (matthewshaver) [inline website/docs/guides/redshift-qs.md]:
```suggestion
Check out [dbt Fundamentals](https://learn.getdbt.com/courses/dbt-fundamentals) for free if you're interested in course learning with videos.
```

**Review by matthewshaver — APPROVED** [review]

File: website/blog/2023-03-30-guide-to-debug-in-jinja.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ date: 2023-03-29
 is_featured: true
 ---
 
-*Editor's note—this post assumes intermediate knowledge of Jinja and macros development in dbt. For an introduction to Jinja in dbt check out [the documentation](https://docs.getdbt.com/docs/build/jinja-macros) and the free self-serve course on [Jinja, Macros, Pacakages](https://courses.getdbt.com/courses/jinja-macros-packages).*
+*Editor's note—this post assumes intermediate knowledge of Jinja and macros development in dbt. For an introduction to Jinja in dbt check out [the documentation](https://docs.getdbt.com/docs/build/jinja-macros) and the free self-serve course on [Jinja, Macros, Pacakages](https://learn.getdbt.com/courses/jinja-macros-and-packages).*
```
Comment (coapacetic) [inline website/blog/2023-03-30-guide-to-debug-in-jinja.md]:
```suggestion
*Editor's note—this post assumes intermediate knowledge of Jinja and macros development in dbt. For an introduction to Jinja in dbt check out [the documentation](https://docs.getdbt.com/docs/build/jinja-macros) and the free self-serve course on [Jinja, Macros, Packages](https://learn.getdbt.com/courses/jinja-macros-and-packages).*
```

File: website/docs/best-practices/dbt-unity-catalog-best-practices.md [hunk 4]
```diff
@@ -61,7 +61,7 @@ Ready to start transforming your Unity Catalog datasets with dbt?
 Check out the resources below for guides, tips, and best practices:
 
 - [How we structure our dbt projects](/best-practices/how-we-structure/1-guide-overview)
-- [Self-paced dbt fundamentals training videos](https://courses.getdbt.com/courses/fundamentals)
+- [Self-paced dbt fundamentals training videos](https://learn.getdbt.com/courses/dbt-fundamentals)
```
Comment (coapacetic) [inline website/docs/best-practices/dbt-unity-catalog-best-practices.md]:
```suggestion
- [Self-paced dbt fundamentals training course](https://learn.getdbt.com/courses/dbt-fundamentals)
```

File: website/docs/docs/build/build-metrics-intro.md [hunk 5]
```diff
@@ -67,5 +67,5 @@ MetricFlow allows you to:
 
 - [Quickstart guide with the dbt Semantic Layer](/guides/sl-snowflake-qs)
 - [The dbt Semantic Layer: what's next](https://www.getdbt.com/blog/dbt-semantic-layer-whats-next/) blog
-- [dbt Semantic Layer on-demand courses](https://courses.getdbt.com/courses/semantic-layer)
+- [dbt Semantic Layer on-demand courses](https://learn.getdbt.com/courses/semantic-layer)
```
Comment (coapacetic) [inline website/docs/docs/build/build-metrics-intro.md]:
```suggestion
- [dbt Semantic Layer on-demand course](https://learn.getdbt.com/courses/semantic-layer)
```

File: website/docs/docs/cloud/about-cloud-setup.md [hunk 6]
```diff
@@ -18,7 +18,7 @@ This portion of our documentation will take you through the various settings in
 
 For steps on installing dbt Cloud development tools, refer to the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation) or the [dbt Cloud IDE (browser-based)](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud).
 
-These settings are intended for dbt Cloud administrators. If you need a more detailed first-time setup guide for specific data platforms, read our [quickstart guides](/guides). If you want a more in-depth learning experience, we recommend taking the dbt Fundamentals on our [dbt Learn online courses site](https://courses.getdbt.com/).
+These settings are intended for dbt Cloud administrators. If you need a more detailed first-time setup guide for specific data platforms, read our [quickstart guides](/guides). If you want a more in-depth learning experience, we recommend taking the dbt Fundamentals on our [dbt Learn online courses site](https://learn.getdbt.com/).
```
Comment (coapacetic) [inline website/docs/docs/cloud/about-cloud-setup.md]:
```suggestion
These settings are intended for dbt Cloud administrators. If you need a more detailed first-time setup guide for specific data platforms, read our [quickstart guides](/guides). If you want a more in-depth learning experience, we recommend taking the dbt Fundamentals on our [dbt Learn site](https://learn.getdbt.com/).
```

File: website/docs/docs/use-dbt-semantic-layer/avail-sl-integrations.md [hunk 7]
```diff
@@ -39,5 +39,5 @@ import AvailIntegrations from '/snippets/_sl-partner-links.md';
 - [dbt Semantic Layer API query syntax](/docs/dbt-cloud-apis/sl-jdbc#querying-the-api-for-metric-metadata) 
 - [Hex dbt Semantic Layer cells](https://learn.hex.tech/docs/logic-cell-types/transform-cells/dbt-metrics-cells) to set up SQL cells in Hex.
 - [Resolve 'Failed APN'](/faqs/Troubleshooting/sl-alpn-error) error when connecting to the dbt Semantic Layer.
-- [dbt Semantic Layer on-demand courses](https://courses.getdbt.com/courses/semantic-layer)
+- [dbt Semantic Layer on-demand courses](https://learn.getdbt.com/courses/semantic-layer)
```
Comment (coapacetic) [inline website/docs/docs/use-dbt-semantic-layer/avail-sl-integrations.md]:
```suggestion
- [dbt Semantic Layer on-demand course](https://learn.getdbt.com/courses/semantic-layer)
```

File: website/docs/guides/azure-synapse-analytics-qs.md [hunk 8]
```diff
@@ -28,7 +28,7 @@ In this quickstart guide, you'll learn how to use dbt Cloud with [Azure Synapse
 - As a Microsoft admin, you’ve enabled service principal authentication. You must add the service principal to the Synapse workspace with either a Member (recommended) or Admin permission set. For details, refer to [Create a service principal using the Azure portal](https://learn.microsoft.com/en-us/entra/identity-platform/howto-create-service-principal-portal) in the Microsoft docs. dbt Cloud needs these authentication credentials to connect to Azure Synapse Analytics.
 
 ### Related content
-- [dbt Courses](https://courses.getdbt.com/collections/beginner)
+- [dbt Courses](https://learn.getdbt.com)
```
Comment (coapacetic) [inline website/docs/guides/azure-synapse-analytics-qs.md]:
```suggestion
- [dbt Learn courses](https://learn.getdbt.com)
```

File: website/docs/guides/bigquery-qs.md [hunk 9]
```diff
@@ -35,7 +35,7 @@ You can check out [dbt Fundamentals](https://courses.getdbt.com/courses/fundamen
 
 ### Related content
 
-- Learn more with [dbt Courses](https://courses.getdbt.com/collections)
+- Learn more with [dbt Courses](https://learn.getdbt.com)
```
Comment (coapacetic) [inline website/docs/guides/bigquery-qs.md]:
```suggestion
- Learn more with [dbt Learn courses](https://learn.getdbt.com)
```

File: website/docs/guides/codespace-qs.md [hunk 10]
```diff
@@ -27,7 +27,7 @@ dbt Labs provides a [GitHub Codespace](https://docs.github.com/en/codespaces/ove
 - [Build your first models](/guides/manual-install?step=3)
 - [Test and document your project](/guides/manual-install?step=4)
 - [Schedule a job](/guides/manual-install?step=5)
-- Learn more with [dbt Courses](https://courses.getdbt.com/collections)
+- Learn more with [dbt Courses](https://learn.getdbt.com)
```
Comment (coapacetic) [inline website/docs/guides/codespace-qs.md]:
```suggestion
- Learn more with [dbt Learn courses](https://learn.getdbt.com)
```

File: website/docs/guides/core-to-cloud-3.md [hunk 11]
```diff
@@ -189,7 +189,7 @@ For the next steps, continue exploring our 3-part-guide series on moving from db
 
 If you need any additional help or have some questions, use the following resources:
 
-- [dbt Cloud learn courses](https://courses.getdbt.com/collections) for on-demand video learning.
+- [dbt Cloud learn courses](https://learn.getdbt.com) for on-demand video learning.
```
Comment (coapacetic) [inline website/docs/guides/core-to-cloud-3.md]:
```suggestion
- [dbt Learn courses](https://learn.getdbt.com) for on-demand video learning.
```

File: website/docs/guides/databricks-qs.md [hunk 12]
```diff
@@ -33,7 +33,7 @@ You can check out [dbt Fundamentals](https://courses.getdbt.com/courses/fundamen
 
 ### Related content
 
-- Learn more with [dbt Courses](https://courses.getdbt.com/collections)
+- Learn more with [dbt Courses](https://learn.getdbt.com)
```
Comment (coapacetic) [inline website/docs/guides/databricks-qs.md]:
```suggestion
- Learn more with [dbt Learn courses](https://learn.getdbt.com)
```

File: website/docs/guides/microsoft-fabric-qs.md [hunk 13]
```diff
@@ -27,7 +27,7 @@ In this quickstart guide, you'll learn how to use dbt Cloud with [Microsoft Fabr
 - As a Microsoft admin, you’ve enabled service principal authentication. You must add the service principal to the Microsoft Fabric workspace with either a Member (recommended) or Admin permission set. For details, refer to [Enable service principal authentication](https://learn.microsoft.com/en-us/fabric/admin/metadata-scanning-enable-read-only-apis) in the Microsoft docs. dbt Cloud needs these authentication credentials to connect to Microsoft Fabric.
 
 ### Related content
-- [dbt Courses](https://courses.getdbt.com/collections)
+- [dbt Courses](https://learn.getdbt.com)
```
Comment (coapacetic) [inline website/docs/guides/microsoft-fabric-qs.md]:
```suggestion
- [dbt Learn courses](https://learn.getdbt.com)
```

File: website/docs/guides/productionize-your-dbt-databricks-project.md [hunk 14]
```diff
@@ -191,7 +191,7 @@ To get the most out of both tools, you can use the [persist docs config](/refere
 
 ### Related docs
 
-- [Advanced deployments course](https://courses.getdbt.com/courses/advanced-deployment) if you want a deeper dive into these topics
+- [Advanced deployments course](https://learn.getdbt.com/courses/advanced-deployment) if you want a deeper dive into these topics
```
Comment (coapacetic) [inline website/docs/guides/productionize-your-dbt-databricks-project.md]:
```suggestion
- [Advanced Deployment course](https://learn.getdbt.com/courses/advanced-deployment) if you want a deeper dive into these topics
```

File: website/docs/guides/redshift-qs.md [hunk 15]
```diff
@@ -32,7 +32,7 @@ You can check out [dbt Fundamentals](https://courses.getdbt.com/courses/fundamen
 
 ### Related content
 
-- Learn more with [dbt Courses](https://courses.getdbt.com/collections)
+- Learn more with [dbt Courses](https://learn.getdbt.com)
```
Comment (coapacetic) [inline website/docs/guides/redshift-qs.md]:
```suggestion
- Learn more with [dbt Learn courses](https://learn.getdbt.com)
```

File: website/docs/guides/sl-migration.md [hunk 16]
```diff
@@ -138,6 +138,6 @@ If you created a new environment in [Step 3](#step-3-setup-the-semantic-layer-in
 - [dbt metrics converter](https://github.com/dbt-labs/dbt-converter)
 - [Why we're deprecating the dbt_metrics package](/blog/deprecating-dbt-metrics) blog post
 - [dbt Semantic Layer API query syntax](/docs/dbt-cloud-apis/sl-jdbc#querying-the-api-for-metric-metadata) 
-- [dbt Semantic Layer on-demand courses](https://courses.getdbt.com/courses/semantic-layer)
+- [dbt Semantic Layer on-demand courses](https://learn.getdbt.com/courses/semantic-layer)
```
Comment (coapacetic) [inline website/docs/guides/sl-migration.md]:
```suggestion
- [dbt Semantic Layer on-demand course](https://learn.getdbt.com/courses/semantic-layer)
```

File: website/docs/guides/snowflake-qs.md [hunk 17]
```diff
@@ -37,7 +37,7 @@ You can also watch the [YouTube video on dbt and Snowflake](https://www.youtube.
 
 ### Related content
 
-- Learn more with [dbt Courses](https://courses.getdbt.com/collections)
+- Learn more with [dbt Courses](https://learn.getdbt.com)
```
Comment (coapacetic) [inline website/docs/guides/snowflake-qs.md]:
```suggestion
- Learn more with [dbt Learn courses](https://learn.getdbt.com)
```

File: website/docs/guides/starburst-galaxy-qs.md [hunk 18]
```diff
@@ -41,7 +41,7 @@ You can also watch the [Build Better Data Pipelines with dbt and Starburst](http
 
 ### Related content
 
-- [dbt Courses](https://courses.getdbt.com/collections)
+- [dbt Courses](https://learn.getdbt.com)
```
Comment (coapacetic) [inline website/docs/guides/starburst-galaxy-qs.md]:
```suggestion
- [dbt Learn courses](https://learn.getdbt.com)
```

**Review by coapacetic — COMMENTED** [review]
final review for copy and links

**Comment by coapacetic** [comment]
heya @dbt-labs/product-docs team -- can I request one more 👍 on this after verifying links and making quick copy changes?

then we can merge away! thank you!

**Pull request merged**